server/text_generation_server/utils/tokens.py

```diff
@@ -125,6 +136,7 @@ def from_pb(
             tokenizer=tokenizer,
             grammar=pb.grammar,
             grammar_type=pb.grammar_type,
+            logit_bias=dict(pb.logit_bias) if pb.logit_bias else None,
```

Comment:
Why is `pb.logit_bias` not possible ? It would maintain consistency better.

Reply:
oh good catch, simplified in the latest commit